.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -61,7 +61,6 @@ body:
   attributes:
     label: Python Version
     description: Attach your version of Python.
-    render: console
```

Comment:
Just use textarea instead. The pre-filled text suggests a multiline output of a command, not just one line of input.

Reply:
Yes, I agree `textarea` will be better.